taxi/views.py

```diff
@@ -1,9 +1,12 @@
+from django.contrib.auth import get_user_model
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
 from django.urls import reverse_lazy
 from django.views import generic
+from django.shortcuts import redirect
 from django.contrib.auth.mixins import LoginRequiredMixin
 
+from .forms import DriverCreationForm, DriverLicenseUpdateForm, CarForm
 from .models import Driver, Car, Manufacturer
```

Comment:
Code Efficiency: You've imported `Driver` model from `.models` but you're not using it in your views. Please replace `Driver` with `get_user_model()` function to get the current user model.